pyomo/contrib/alternative_solutions/tests/test_gurobi_lp_enum.py

```diff
@@ -31,15 +30,20 @@
 @unittest.skipUnless(numpy_available, "NumPy not found")
 class TestLPEnumSolnpool(unittest.TestCase):
 
+    def test_non_positive_num_solutions(self):
+        """
+        Confirm that an exception is thrown with a non-positive num solutions
+        """
+        n = tc.get_pentagonal_pyramid_mip()
+        with self.assertRaises(AssertionError):
+            gurobi_enumerate_linear_solutions(n, num_solutions=-1)
```

Comment:
This should be a `ValueError`, I think. Though what matters most is that it *is* an error! :)